<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="main_layout" stretch="6,0,6">
      <item>
       <layout class="QVBoxLayout" name="left_pane" stretch="12">
        <item>
         <widget class="QListWidget" name="plugins">
          <property name="minimumSize">
           <size>
            <width>260</width>
            <height>480</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="flow_buttons" stretch="1,0,0,4">
        <item>
         <spacer name="top_space">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="flowPluginAdd">
          <property name="text">
           <string>-&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="flowPluginRemove">
          <property name="text">
           <string>&lt;-</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="bottom_space">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="right_pane" stretch="4,3">
        <item>
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Sunken</enum>
          </property>
          <layout class="QGridLayout" name="flow_container">
           <item row="0" column="0">
            <widget class="QListWidget" name="flow_list"/>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>2</number>
          </property>
          <widget class="QWidget" name="tab_Settings">
           <attribute name="title">
            <string>Settings</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <layout class="QVBoxLayout" name="settings_container"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_Output">
           <attribute name="title">
            <string>Output</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QTextEdit" name="flow_output">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_Queue">
           <attribute name="title">
            <string>Queue</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QListWidget" name="listWidget"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_flow"/>
    <addaction name="actionOpen_flow"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="pluginAdd"/>
   <addaction name="pluginRemove"/>
   <addaction name="runFlow"/>
  </widget>
  <action name="actionNew_flow">
   <property name="text">
    <string>New flow</string>
   </property>
  </action>
  <action name="actionOpen_flow">
   <property name="text">
    <string>Open flow</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="pluginAdd">
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Add a new plugin</string>
   </property>
  </action>
  <action name="pluginRemove">
   <property name="text">
    <string>Remove</string>
   </property>
   <property name="toolTip">
    <string>Remove an existing plugin</string>
   </property>
  </action>
  <action name="runFlow">
   <property name="text">
    <string>&amp;Run</string>
   </property>
   <property name="toolTip">
    <string>Run the current flow script</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
